### PR TITLE
trial addition

### PR DESCRIPTION
**Pull request by seabbs**
This PR adds a few packages either in dev or linked to papers. Let me know what else is needed here!

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -31,4 +31,12 @@
     "package": "stackr",
     "url": "https://github.com/epiforecasts/stackr"
   }
```
Comment (Bisaloo) [inline packages.json]:
```suggestion
  },
```

File: packages.json [hunk 2]
```diff
@@ -31,4 +31,12 @@
...
+  {
+    "package": "idbrms",
+    "url": "https://github.com/epiforecasts/idbrms"
+  }
```
Comment (Bisaloo) [inline packages.json]:
```suggestion
  },
```

File: packages.json (hunk 1 above)
Comment (seabbs) [reply packages.json]: 🙈

**Comment by seabbs** [comment]
Ooops thanks 😄

**Review by Bisaloo — APPROVED** [review]

**Pull request merged**